src/store/todoSlice/todoSlice.test.js

```diff
@@ -0,0 +1,151 @@
+import reducer, {
+  addTodo,
+  deleteTodo,
+  editTodo,
+  toogleTodo,
+} from "./todoSlice";
+
+test("should handle a todo being added to an empty list", () => {
```

Comment:
move all tests related to common context to `describe` statement. And write `it` for each test case